scripts/index.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
 
 import React from 'react';
-import router from './router';
+import router, {transitionTo} from './router';
 
 router.run((Handler, state) => {
-  React.render(<Handler {...state} />, document.body);
+  // I know, passing `transitionTo` like this is ugly,
+  // haven't found a better solution yet though.
+  // Since `Explore` component is not in the Router `context`
+  // we have to pass it along, somehow.
+  React.render(<Handler {...state} transitionTo={transitionTo} />, document.body);
```

Comment:
What do you mean by “not in the Router context”? You should be able to declare `contextTypes: { router: PropTypes.func }` in `Explore` and have access to `this.context.router` from there.